.github/workflows/botonic-core.tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Botonic core tests
+ 
+on: [push]
+ 
+jobs:
+  botonic_react_tests:
+    name: Botonic core tests
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checking out to current branch (Step 1 of 6)
+      uses: actions/checkout@v1
+    - name: Setting up node (Step 2 of 6)
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12'
+    - name: Setting up cache (Step 3 of 6)
+      uses: actions/cache@v1
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+    - name: Install botonic-core (Step 4 of 6)
+      run: (cd ./packages/botonic-core && npm install)
+    - name: Build botonic-core (Step 5 of 6)
+      run: (cd ./packages/botonic-core && npm run build)
```

Comment:
all workflows are very similar. 
And we have still need to add other plugins (watson, nlu, segment,...)
In a separate PR I'd investigate if we can reuse a common workflow to avoid repetitions. Maybe with
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsuses ?